ruby/ruby_trunk/discourse_benchmarks/Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM rubybench/ruby:0.3
 MAINTAINER Guo Xiang "tgx_world@hotmail.com"
+RUN apt-get update
```

Comment:
do `apt -get update && apt-get install wget` instead. Don't need to add a new RUN command